app/views/admin/events/index.html.erb

```diff
@@ -16,9 +17,14 @@
     </tr>
   </thead>
 
-  <tbody>
+  <tbody data-role="activerecord_sortable">
```

Comment:
@bitTal can you revert the changes made to the views other than the campaigns, please?